core/gui/src/app/dashboard/component/user/cluster/cluster.component.ts

```diff
@@ -113,4 +113,23 @@ export class ClusterComponent implements OnInit {
   closeClusterManagementModal(): void {
     this.isClusterManagementVisible = false;
   }
+
+  getBadgeStatus(status: string): string[]{
+    switch(status){
+      case "LAUNCHING":
+      case "LAUNCHED":
+      case "RESUMING":
+        return ["play-circle", "orange"];
```

Comment:
It would be beneficial to differentiate the color or icon between the `LAUNCHING` and `LAUNCHED` statuses, as well as between `TERMINATING` and `TERMINATED`. 

Additionally, the list of statuses mentioned here differs from the table schema. The current status options in the schema are `ENUM('LAUNCHING', 'LAUNCHED', 'PAUSING', 'PAUSED', 'RESUMING', 'TERMINATING', 'TERMINATED')`. Could we ensure consistency across both?

Reply:
Okay, I just pushed changes that make all clusters in the process of launching, resuming, and terminating to be orange. Otherwise, launched is green and terminated or failed are both red. Furthermore, I removed the status for "Completed".

Reply:
Looks good! Thank you.